plugins/available/pretty-cat.plugin.bash

```diff
@@ -0,0 +1,21 @@
+cite about-plugin                                                                                        
+about-plugin 'pygmentize instead of cat to terminal if possible'                                         
+
+if [ -z $(which pygmentize) ]
+then
+    echo "Pygments is required to use this pluging"
+    echo "Install it by doing 'pip install Pygments' as the superuser"
+    exit 1
```

Comment:
Do you really want to `exit` here? This would exit the actual shell/Bash instance, and it would prevent people from opening a new Bash instance.

Using `exit` is probably a bit drastic in this case...